### DISCUSSION TITLE
Feature: Shutdown from Web GUI

### DISCUSSION THREAD

**Discussion by Peeeet** [IceWhaleTech/CasaOS#253] [Ideas]
### Is your idea related to a problem? Please describe:
I don't see an obvious way to safely shutdown through CasaOS.  Specifically when running on the ZimaBoard... the word "shutdown" is not mentioned anywhere and this, in fact, will be the first thing on this github that even contains the word "shutdown" 🧐
<br>
### Describe the solution you'd like:
A button in the Web UI that shuts down the underlying system (in my case on a ZimaBoard).
<br>
### Describe alternatives you've considered:
SSH is an option, but for such a rudimentary feature, I feel it should be as accessible as so many of the other more advanced features that are already present in the Web UI.

...Unless this system / the ZimaBoard are built to gracefully handle unexpected shutdowns (aka just pullin' the ol' plug) in which case that's an awesome feature and should be ~documented somewhere~ advertised proudly!
<br>
### Additional context
Also this idea is even more pertinent considering the current [bug 244](https://github.com/IceWhaleTech/CasaOS/issues/244) that prevents SSH login from the Web UI 😅

**Comment by AuthorShin** [comment]
Also having Reboot too would be nice.

**Comment by Lauren-ED209** [comment]
I've transfer this request to development schedule for further discussion! Thanks for your suggestion!

**Comment by tigerinus** [comment]
@ETWang1991 FYI
